.github/workflows/bucket-cleanup.yml

```diff
@@ -12,6 +12,7 @@ jobs:
     env:
       GOPATH: ${{ github.workspace }}/go
     name: Bucket cleanup
+    environment: production
```

Comment:
ahh yeah sure @cnunciato . This specifies the Github environment to use to reference specific variables for that environment. This is configured here in GitHub in this repo under settings. I created a `testing` environment and a `production` environment which contains the environment variables it should use/override for the environment specified. As an example if you specify `environment: testing` it would set `PULUMI_STACK_NAME` to `pulumi/www-testing`  and if you specified production in the workflow it would set that evironment variable to the productions stack. You can see the configuration under the repo setttings and then going to environment on the left nav.